src/components/dashboard/Claim.js

```diff
@@ -38,13 +36,13 @@ const Claim = props => {
   const { screenProps, styles }: ClaimProps = props
   const store = SimpleStore.useStore()
   const gdstore = GDStore.useStore()
-
+  const { entitlement } = gdstore.get('account')
   const [showDialog] = useDialog()
   const [loading, setLoading] = useState(false)
   const [claimInterval, setClaimInterval] = useState(null)
   const [state, setState]: [ClaimState, Function] = useState({
     nextClaim: '--:--:--',
-    entitlement: 0,
+    entitlement: entitlement || 0,
```

Comment:
You have entitlement from store account and entitlement state...I think it would be better if we use different names for them because it could be confusing...